Library/Formula/tokyo-tyrant.rb

```diff
@@ -15,6 +15,11 @@ class TokyoTyrant < Formula
     patch :DATA
   end
 
+  patch :p0 do
+    url 'https://gist.githubusercontent.com/actsasflinn/348633/raw/32561ebf6487650ce86d0184f0ca35bb8110d04c/macosx_snow_leopard_socket_fix.patch'
+    sha1 '2b73fe52c37cd41b8e13112c0bc083bce8ae3c9e'
+  end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!